joomla/joomla-cms
# Option to stop robots inc banners views

Pull Request for Issue # .

### Summary of Changes
Option to stop increment count of banner views if search engine is detected. Count views as before by default for BC.


### Testing Instructions
Coder review
OR
1. Create banner and open page with this banner. Check if banner view count increment.
2. Set option "Robots Increment Impressions" to "NO"
3. Reload page with banner. Check if banner impressions count increment. Must be +1.
4. Change user-agent to anything includes "Googlebot".
5. Reload page with banner. Check if banner impressions count increment. Must be same as before.
PS: Its can be tested local too. For example in Firefox you can open Developer tools and start Responsive/Mobile view, then add new Device with any user agent.

### Expected result
If set option "Robots increment impressions" to "NO", Module should not increment banner views if search engine detected


### Actual result
Module increments count of banner impressions always


### Documentation Changes Required
No?
